### ISSUE TITLE
Images don't contain newest JupyterLab pointrelease

### ISSUE THREAD

**Issue by nils-werner** [jupyter/docker-stacks#716]
JupyterLab was [updated to `0.34.7` a few days ago](https://github.com/jupyterlab/jupyterlab/releases), and the latest Docker image `jupyter/scipy-notebook:137a295ff71b` was built a few hours ago. The build instructions for this image contain the lines

    RUN conda install --quiet --yes \
        [...]
        'jupyterlab=0.34.*' && \
        [...]

So I was expecting to see the latest pointrelease of JupyterLab in the image, but it is still on `0.34.0`

I was looking forward to JupyterLab 0.34.6 or newer because it [introduced a "Quit" button](https://github.com/jupyterlab/jupyterlab/issues/4414) that allowed people to shut down the entire Jupyter instance (not just the kernel). Such a button is particularly handy when people want to switch images in a multi-user JupyterHub installation.

I can't find any Docker build logs, so I cannot debug what the issue here is myself. Does anyone have an idea or any pointers to the build logs?

**What docker image you are using?**

Example: `jupyter/scipy-notebook:137a295ff71b`

**What complete docker command do you run to launch the container (omitting sensitive values)?**

Example: `docker run -it --rm jupyter/scipy-notebook:137a295ff71b /bin/bash`

**What steps do you take once the container is running to reproduce the issue?**

 1. Run `conda list`

**What do you expect to happen?**

See a line

    jupyterlab                0.34.7                   py36_0    conda-forge

**What actually happens?**

I see a line

    jupyterlab                0.34.0                   py36_0    conda-forge

**Comment by parente** [comment]
Not sure if you can see https://cloud.docker.com/swarm/jupyter/repository/registry-1.docker.io/jupyter/base-notebook/builds/7c4aa0e1-8db3-4c2e-846e-91e21eba22b9, but it appears Docker Cloud used cached base-notebook layers for this build. That's not behavior I remember seeing before. I'll have to investigate if there's a way to disable that caching mechanism.

**Comment by nils-werner** [comment]
I can't see the logs, sorry.

Just a wild guess: Are all image builds started simultaneously? In that case each `Dockerfile` inheriting `FROM base-notebook` might use last cycle's version of that image...

**Comment by parente** [comment]
That's correct. I've not seen the caching like this on Docker Cloud before. It's either new or I missed it in the past.

A PR that specifies the exact point release of the packages of importance in base-notebook will fix the problem. It'll trade of automatically upgrading to the latest point release when there's other changes in base-notebook, but those have been late, and will continue, to be few and far between.

**Comment by ericdill** [comment]
@parente can be closed since #737 was merged

@nils-werner -- the latest version of docker images from docker-stacks has been updated to include jupyterlab 0.34.12
